web-ui/src/context/SkillsContext.js

```diff
@@ -7,6 +7,7 @@ export const MY_SKILL_TOGGLE = "toggle";
 export const MY_PROFILE_UPDATE = "update";
 export const UPDATE_PDL = "update_pdl";
```

Comment:
SkillsContext is a placeholder right now that will be reworked and split up later? 

Holding default data so you don't have to mess around with inserting fake data?

I want to know your approach here, because I'm looking at working on a component for the FE.

Thanks